specs/deneb/polynomial-commitments.md

```diff
@@ -222,44 +223,33 @@ def blob_to_polynomial(blob: Blob) -> Polynomial:
     return polynomial
 ```
 
-#### `compute_challenges`
+#### `compute_challenge`
 
 ```python
-def compute_challenges(polynomials: Sequence[Polynomial],
-                       commitments: Sequence[KZGCommitment]) -> Tuple[Sequence[BLSFieldElement], BLSFieldElement]:
+def compute_challenge(blob: Blob,
+                      commitment: KZGCommitment) -> BLSFieldElement:
     """
     Return the Fiat-Shamir challenges required by the rest of the protocol.
```

Comment:
```suggestion
    Return the evaluation point challenge using Fiat-Shamir.
```

Reply:
Well I don't think that wording is correct. Fiat-Shamir is a transformation (turning interactive protocols into non-interactive ones) not a way to compute challenges.

Reply:
Good point. I see it was changed to "challenge" which is accurate so we can disregard this.